feature-libs/cart/quick-order/components/quick-order/form/quick-order-form.component.ts

```diff
@@ -42,20 +44,23 @@ export class QuickOrderFormComponent implements OnInit, OnDestroy {
     }
 
     event?.preventDefault();
-
+    this.isSearching = true;
```

Comment:
what's the point of isSearching?

Reply:
Add button in the form is disabled when flag 'isSearching' is true.

Reply:
good point!

Reply:
example:


https://user-images.githubusercontent.com/29099466/127384081-ba32cd02-2090-4022-b5a7-347914901039.mov

Reply:
example - no throttling 


https://user-images.githubusercontent.com/29099466/127384345-1ce4b419-3ce1-40c9-8afe-fed9c5ba7418.mov

Reply:
good point